src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -189,7 +200,40 @@ public void showThereIsOnlyOneAccount() {
     }
     //@@author Sherlock-YH
     public void showMainAccountSwitched() {
-        System.out.println("Main Account switched");
         printLine();
```

Comment:
Print the line before this function is called, don't printLine() as the first thing inside the function. Standardisation reasons